airflow/models/dag.py

```diff
@@ -1621,6 +1621,13 @@ def sync_to_db(self, sync_time=None, session=None):
         """
         self.bulk_sync_to_db([self], sync_time, session)
 
+    def get_default_view(self):
+        """This is only there for backward compatible jinja2 templates"""
+        if self.default_view is None:
+            return conf.get('webserver', 'dag_default_view').lower()
+        else:
+            return self.default_view
```

Comment:
```suggestion
    @property
    def default_view(self):
        """This is only there for backward compatible jinja2 templates"""
        if self._default_view is None:
            return conf.get('webserver', 'dag_default_view').lower()
        else:
            return self._default_view
    @default_view.setter
    def default_view_setter(self, value):
        self._default_view= value
```
This way we won't have two similar public attributes of default_view as a property, get_default_view as a method.

Reply:
I can remove this piece tbh, I just added it so that it is in parity with DAGModel. Will remove it

Reply:
I kept it there for parity between Dag and DagModel.

The other option is to create a Database migration to fill all the `null` values, WDYT? @mik-laj

Reply:
@mik-laj Any comments?

Reply:
Okey. Now I understand the whole change. It looks good.